Wikipedia/Code/Session.swift

```diff
@@ -280,10 +279,10 @@ import Foundation
                 completionHandler(nil, response, resultParsingError)
             }
         }) else {
+            completionHandler(nil, nil, RequestError.invalidParameters)
```

Comment:
should this also be added to `jsonCodableTask`? when it returns nil on line `256`

Reply:
Yeah, good catch